consensus-types/blocks/roblock.go

```diff
@@ -0,0 +1,63 @@
+package blocks
+
+import (
+	"sort"
+
+	"github.com/pkg/errors"
+	fieldparams "github.com/prysmaticlabs/prysm/v4/config/fieldparams"
+	"github.com/prysmaticlabs/prysm/v4/consensus-types/interfaces"
+)
+
+var ErrRootLength error = errors.New("incorrect length for hash_tree_root")
+
+type ROBlock struct {
```

Comment:
I don't like this name. It's very cryptic. Also there is no doc.

Reply:
we can address this change in the `eip4844` branch, and if we ever move this onto the `develop` branch